packages/app-utils/src/components/ConnectionBootstrap.tsx

```diff
@@ -29,8 +30,10 @@ export function ConnectionBootstrap({
 }: ConnectionBootstrapProps): JSX.Element {
   const api = useApi();
   const client = useClient();
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const [error, setError] = useState<unknown>();
```

Comment:
If it's unused then remove it. But looks like this should be used still to display when the server shuts down